client/question/services/code_preprocessors/PythonCodePreprocessorService.js

```diff
@@ -313,6 +313,28 @@ tie.factory('PythonCodePreprocessorService', [
       return VALID_WHITESPACE_REGEX.test(c);
     };
 
+    /**
+    * Creates and sets the string separator for user print stdOut.
+    *
+    * @returns {string}
+    * @private
+    */
+    var _generateStringSeparator = function() {
```

Comment:
_generateStdoutSeparator perhaps -- "string separator" is quite generic.

Also, there is a mismatch between function name and implementation. The function sounds like it's supposed to generate a "string separator", but instead returns something called separatorCode...

Reply:
Fixed.